buildout.cfg

```diff
@@ -8,19 +8,3 @@ versions = versions
 [sftpcopy]
 recipe = zc.recipe.egg:scripts
 eggs = gocept.sftpcopy
-
-[test]
-recipe = zc.recipe.egg
-scripts = py.test=test
-eggs = gocept.sftpcopy[test,amqp,test_amqp]
-    pytest
-    pytest-sugar
-    pytest-cov
-    pytest-flake8
-    gocept.pytestlayer
-
-[versions]
```

Comment:
As we removed the `[test]` and `[versions]` sections it might be at least a good idea to remove the respecetive references in the `[buildout]` section or delete the file completely.

Reply:
Okay, as only the entry point is left now, it no longer makes any sense to keep `buildout.cfg`. Installation of the console script can be done via pip nowadays.